### PR TITLE
FIxes #583

### LINKED ISSUE #583: Rectpacking does not reduce initially estimated width.
Screenshot from 2020-04-24 08-56-00
```
@layout [aspectRatio] 1.6
scchart Example {
  
  
  region A {
    initial state A
  }
  
  region B {
    initial state A
  }
  
  region C {
    initial state A
  }
  
  region D {
    initial state A
  }
  
  region E {
    initial state A
    go to B
    go to C
    state B
    go to A
    state C
    go to D
    state D
  }
  
  region F {
    initial state A
  }
  
  region G {
    initial state A
  }
  
  region H {
    initial state A
    go to B
    go to C
    state B
    go to A
    state C
    go to D
    state D
  }
  
  region I {
    initial state A
  }
  
  region J {
    initial state A
  }
  region K {
    initial state A
  }
  
  region L {
    initial state A
  }
}
```

### PR DESCRIPTION
**Pull request by soerendomroes**
Signed-off-by: Soeren Domroes <sdo@informatik.uni-kiel.de>

### CODE REVIEW

**Comment by soerendomroes** [comment]
No, I implemented this bug specifically for this algorithm.

**Comment by uruuru** [comment]
Sorry in advance but: you implemented a bug 🤔?

**Comment by soerendomroes** [comment]
I disguised it as a feature first. I will add unit tests to prevent this in the future.

File: test/org.eclipse.elk.rectpacking.test/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: ELK Rectpacking Algorithms Tests
+Bundle-SymbolicName: org.eclipse.elk.alg.rectpacking.test;singleton:=true
+Bundle-Version: 0.7.0.qualifier
+Automatic-Module-Name: org.eclipse.elk.alg.common.test
```
Comment (uruuru) [inline test/org.eclipse.elk.rectpacking.test/META-INF/MANIFEST.MF:6]: Is this correct?

File: test/org.eclipse.elk.rectpacking.test/src/org/eclipse/elk/alg/rectpacking/issues/Issue583Test.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0  *******************************************************************************/
```
Comment (uruuru) [inline test/org.eclipse.elk.rectpacking.test/src/org/eclipse/elk/alg/rectpacking/issues/Issue583Test.java:8]: Formatting is broken.

File: test/org.eclipse.elk.rectpacking.test/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: ELK Rectpacking Algorithms Tests
+Bundle-SymbolicName: org.eclipse.elk.alg.rectpacking.test;singleton:=true
+Bundle-Version: 0.7.0.qualifier
+Automatic-Module-Name: org.eclipse.elk.alg.rectpacking.test
```
Comment (le-cds) [inline test/org.eclipse.elk.rectpacking.test/META-INF/MANIFEST.MF:6]: Doesn't match the plug-in folder, so.. the latter should be renamed.

**Pull request merged**